modules/openy_gc_auth/modules/openy_gc_auth_custom/src/Plugin/GCIdentityProvider/Custom.php

```diff
@@ -24,6 +24,11 @@ public function defaultConfiguration():array {
     return [
       'enable_recaptcha' => TRUE,
       'api_endpoint' => '/openy-gc-auth/provider/custom/login',
+      'enable_email_verification' => TRUE,
+      'email_verification_api_endpoint' => '/openy-gc-auth/provider/custom/login-by-link',
+      'email_verification_link_life_time' => 14400,
```

Comment:
class const? - a must
config value with a form? - an option for future

Reply:
Fixed